### PR TITLE
Modification of input distribution for firesatellite use case

### LINKED ISSUE #2527: Strange probabilistic model for the fire satellite use-case
### What happened?

The probabilistic model of the fire satellite use-case (https://openturns.github.io/openturns/latest/usecases/use_case_fireSatellite.html) is made of independent normal distributions. It allows to produce negative values for quantities which are physically constrained to be positive. Many of these normal distributions have a standard deviation much lower than their mean value, resulting into a numerical truncation to positive values, but others like the reflectance factor or the drag coefficient can take negative values with significant probabilities.

### How to reproduce the issue?

```shell
Read the use-case description
```

### Version

1.23dev

### Operating System

Linux

### Installation media

from source

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by m-balesdent**
Hello,

Closes  #2527

This PR proposed a modification of the probability distribution of input parameters of the firesatellite use case, to avoid non realistic samples (negative for positive quantities). 

Truncated  Normal distribution is used instead of Normal distribution.

### CODE REVIEW

**Review by jschueller — COMMENTED** [review]
its not always necessary to truncate when sigma is small wrt to the mean
(when the numerical range does not contain negative values)
but it doesnt hurt either

note that the body of the commit should contain "Closes #xxx" (instead of the commit title)
+1

File: python/doc/usecases/use_case_fireSatellite.rst [hunk 1]
```diff
@@ -21,25 +21,25 @@ The multidisciplinary analysis that is used to compute the converged values of t
     :width: 50%
 
 
-This test case is composed of nine random variables:
+This test case is composed of nine random variables distributed according to truncated Normal distributions :math:`\mathcal{TN}(\mu,\sigma,a,b)` with :math:`\mu` the mean, :math:`\sigma` the standard deviation, :math:`a` the lower bound and :math:`b` the upper bound:
```
Comment (jschueller) [inline python/doc/usecases/use_case_fireSatellite.rst:24]: skip lines, its a bit long

**Comment by m-balesdent** [comment]
> its not always necessary to truncate when sigma is small wrt to the mean (when the numerical range does not contain negative values) but it doesnt hurt either
> 
> note that the body of the commit msg should contain "Closes #xxx" (instead of the commit title) +1

The commit msg has been changed.

Concerning the truncated distribution, it could be more consistent to keep the same distribution for all the variables.

**Review by josephmure — APPROVED** [review]

File: python/src/usecases/fireSatellite_function.py [hunk 2]
```diff
@@ -156,31 +156,31 @@ def __init__(self):
...
-        self.RD = ot.Normal(5.0, 1.0)
+        self.RD = ot.TruncatedNormal(5.0, 1.0, 5.0 - 3 * 1.0, 5.0 + 3 * 1.0)
 
         # Moment arm for aerodynamic torque
-        self.Lalpha = ot.Normal(2.0, 0.4)
+        self.Lalpha = ot.TruncatedNormal(2.0, 0.4, 2.0 - 3 * 0.4, 2.0 + 3 * 0.4)
 
         # Drag coefficient
-        self.Cd = ot.Normal(1.0, 0.3)
+        self.Cd = ot.TruncatedNormal(1.0, 0.3, 1.0 - 3 * 0.3, 1.0 + 3 * 0.3)
```
Comment (regislebrun) [inline python/src/usecases/fireSatellite_function.py:183]: positive, but it was close ;-)

**Review by regislebrun — APPROVED** [review]

**Pull request merged**